src/ui/dfn-list.js

```diff
@@ -49,9 +49,33 @@ function show() {
           <a href="${`#${dfn.id}`}">
             ${dfn.textContent}
           </a>
+          ${labelDfnIfExported(dfn)} ${labelDfnIfUnused(dfn)}
         </li>
       `;
     });
   render`${definitionLinks}`;
   ui.freshModal(l10n.list_of_definitions, ul, button);
 }
+
+/**
+ * If a definition is exported, label it accordingly
+ * @param {HTMLElement} dfn a definition
+ */
+function labelDfnIfExported(dfn) {
+  if (dfn.hasAttribute("data-export")) {
+    return hyperHTML`<span class="exported">(exported)</span>`;
```

Comment:
Maybe remove the parenthesis? 
```suggestion
    return hyperHTML`<span class="exported">exported</span>`;
```

Reply:
imho keeping the parenthesis is good in case the styling doesn't jive with whatever the user is using to read the specs. otherwise they might think that `exported` or `unused` is part of the definition name.

Reply:
I don't have a strong opinion on this. Happy to keep them, happy to remove them. 
BTW, if styling didn't work, most likely JS didn't work either.

Reply:
hm. also thinking of colour-blindness and stuff. for example, there's a type of colour-blindness called `Achromatopsia` where vision is essentially monochromatic. folk like that would probably appreciate the parentheses.

Reply:
> hm. also thinking of colour-blindness and stuff. for example, there's a type of colour-blindness called `Achromatopsia` where vision is essentially monochromatic. folk like that would probably appreciate the parentheses.

Does the same apply for GitHub labels? Does GitHub do something for them?

Reply:
@saschanaz interesting question. i don't know!

Reply:
Can we go without parentheses for now? Because:

* Wikipedia says total color blindness is not just a normal colorblind but a more serious medical syndrome.
* Even if someone is total colorblind they should still be able to see gray background

So for now I don't think additional visual indication is needed.